src/main/java/io/confluent/connect/jdbc/source/BulkTableQuerier.java

```diff
@@ -36,8 +36,8 @@
 public class BulkTableQuerier extends TableQuerier {
   private static final Logger log = LoggerFactory.getLogger(BulkTableQuerier.class);
 
-  public BulkTableQuerier(QueryMode mode, String name, String topicPrefix) {
-    super(mode, name, topicPrefix);
+  public BulkTableQuerier(QueryMode mode, String name, JdbcSourceTaskConfig config) {
```

Comment:
I think it would be cleaner to just add a boolean parameter for the new config property... the other parameters that remain are also derived from the config, so that feels more consistent.